announcements/1.16.0.md

```diff
@@ -0,0 +1,100 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.16.0.
+
+Please see the [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Android
+
+#### Android 13 support
+
+Google officially released Android 13 in August 2022, and it's now fully available as part of Anbox Cloud. You can find a general overview of the changes and new features in the [official Android documentation](https://developer.android.com/about/versions/13). Android 13 is supported with the same set of features in Anbox Cloud as all other Android versions.
+
+#### Serial ID
+
+Android typically provides a hardware-derived serial number for the system in the `ro.serialno` system property. Anbox Cloud never set the property to a particular value, but with this release, we set a value derived from the container ID. This method fulfils Android's conformance requirements.
+
+### AMD and Intel GPU support
+
+GPU support for AMD and Intel now uses the respective Mesa Vulkan drivers ([`radv`](https://docs.mesa3d.org/drivers/radv.html),  [`anv`](https://docs.mesa3d.org/drivers/anv.html)) with [ANGLE](https://chromium.googlesource.com/angle/angle) as the EGL/OpenGL ES frontend driver.
+
+[Mesa](https://mesa3d.org/) was updated to the [22.2.2](https://docs.mesa3d.org/relnotes/22.2.2.html) release, which provides various improvements and bug fixes.
+
+### Anbox runtime
+
+#### Split of Android's system and vendor directories
+
+With the introduction of support for Android 13, Anbox Cloud now separates the `/system` and `/vendor` partitions of Android into separate directories. Inside the LXD containers, the two partitions are available at `/var/lib/anbox/android-system` and `/var/lib/anbox/android-vendor`. From a user perspective nothing changes though, because both directories are combined at `/var/lib/anbox/rootfs`. Therefore, no changes to existing addons or customisations are required. The split allows to use custom Android system images in a future Anbox Cloud release.
+
+#### New bug report utility
+
+The Anbox [images](https://anbox-cloud.io/docs/ref/provided-images) include a new bug report tool called `anbox-bug-report`, which is extensible and collects information in an archive rather than a file. This method allows developers and support to easier inspect the information when debugging issues. See the [documentation](https://anbox-cloud.io/docs/howto/troubleshoot/bug-report) for more details.
+
+### WebRTC streaming improvements
+
+To further allow better fine-tuning of the WebRTC based streaming implementation in Anbox Cloud, 1.16 includes various small improvements:
```

Comment:
```suggestion
To allow better fine-tuning of the WebRTC based streaming implementation in Anbox Cloud, 1.16 includes various small improvements:
```

Reply:
Either "further" or "better", but not both. ;)

Reply:
You're right, I misread your suggested changes!